client/app/bundles/course/users/components/UserProfileCard.tsx

```diff
@@ -0,0 +1,193 @@
+import PropTypes from 'prop-types';
+import { FC } from 'react';
+import {
+  Avatar,
+  Box,
+  Card,
+  CardContent,
+  Grid,
+  Link,
+  Paper,
+  Typography,
+} from '@mui/material';
+import { scroller } from 'react-scroll';
+import { blue, green, orange } from '@mui/material/colors';
+
+interface Props {
```

Comment:
So if you pass the user instead, you can use UserEntity as the type of the props instead of re-declaring the interface